src/xyObject/xyObjectBestPoints.ts

```diff
@@ -11,23 +11,27 @@ import { xyObjectMinXPoint } from './xyObjectMinXPoint';
  * new property `close` to true
  *
  * @param points - array of all the points
- * @param [options={}] Options
- * @param [options.from] - min X value of the window to consider
- * @param [options.to] - max X value of the window to consider
- * @param [options.limit=20] - max number of points
- * @param [options.threshold=0.01] - minimal intensity compare to more intense point
- * @param [options.numberSlots=10] - define the number of slots and indirectly the slot width
- * @param [options.numberCloseSlots=50] numberCloseSlots
+ * @param options - Options
+ * @param options.from - min X value of the window to consider
+ * @param options.to - max X value of the window to consider
+ * @param options.limit - max number of points
+ * @param options.threshold - minimal intensity compare to more intense point
+ * @param options.numberCloseSlots - numberCloseSlots
+ * @param options.numberSlots - define the number of slots and indirectly the slot width
  * @returns - copy of points with 'close' property
  */
 export function xyObjectBestPoints(
   points: Point[],
   options: {
     from?: number;
     to?: number;
+    /**@default 20 */
     limit?: number;
+    /**@default 0.01 */
```

Comment:
It seems to me that the explanation of the parameter should also be at this level and the @param from the documentation should disappear. Please check if the resulting generated documentation that the result is ok.

Reply:
okey thank you
i checked the generated documentation and the result is ok